packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -100,12 +101,18 @@ class AddressDetails extends React.Component {
                                     height_offset='179px'
                                     is_disabled={isDesktop()}
                                 >
-                                    <p className='details-form__description'>
+                                    <Text
+                                        as='p'
+                                        align='left'
```

Comment:
Isn't left a default value for `align`?

Reply:
Actually, the default is center. I tested again.

Reply:
So the first thing that comes to mind is **why**? A default case is a case which is supposed to support most frequent cases first. Is `center` more frequent than `left` in our code base?

Reply:
I think we should follow default values set in the CSS spec:
As per link [here](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align):
`Initial value | start, or a nameless value that acts as left if direction is ltr, right if direction is rtl if start is not supported by the browser.`




Initial value | start, or a nameless value that acts as left if direction is ltr, right if direction is rtl if start is not supported by the browser.
-- | --




Initial value | start, or a nameless value that acts as left if direction is ltr, right if direction is rtl if start is not supported by the browser.
-- | --